modules/dr-restoring-cluster-state.adoc

```diff
@@ -298,9 +298,7 @@ csr-4hl85   13m    kubernetes.io/kube-apiserver-client-kubelet   system:servicea
 csr-zhhhp   3m8s   kubernetes.io/kube-apiserver-client-kubelet   system:serviceaccount:openshift-machine-config-operator:node-bootstrapper   Pending <2>
 ...
 ----
-<1> A pending kubelet service 
-
-CSR (for user-provisioned installations).
+<1> A pending kubelet service CSR (for user-provisioned installations).
```

Comment:
I know you were only fixing the typo here but I wanted to point out IBMSG suggestion to avoid using parentheses in general text. https://www.ibm.com/docs/en/ibm-style?topic=punctuation-parentheses#general-text

Reply:
Actually, the text in parenthesis should be fully removed. However, that would imply a technical change that requires QE and it would be pretty important to fix this bug ASAP (the destroyed list makes the procedure difficult to follow).

So @skrthomas would it work if we just merge this as-is and I open a follow-up bug to remove this?

Reply:
Yes, please open a follow-up bug to confirm the removal with QE.

Reply:
I was actually just suggesting to remove the parentheses. Could it instead be:
> <1> A pending kubelet service CSR for user-provisioned installations. 

So all you're removing is the (), and I don't think that would require further QE approval.

Reply:
But nvm its fine as is. Just a suggestion and unrelated to the change at hand anyways. Its fine merged as is, no worries :)

Reply:
Follow-up bug: https://issues.redhat.com/browse/OCPBUGS-31247

I'll just remove the parenthesis and make some generic improvement to that part.